tests/core/admin-module/test_admin_nodeInfo.py

```diff
@@ -1,7 +1,20 @@
+import pytest
+
+
 def test_admin_nodeInfo(web3, skip_if_testrpc):
     skip_if_testrpc(web3)
 
-    node_info = web3.geth.admin.nodeInfo
+    with pytest.warns(DeprecationWarning):
+        node_info = web3.geth.admin.nodeInfo
+
+        assert 'enode' in node_info
```

Comment:
Random idea: We end up wanting to talk about enode addresses in web3.py, trinity, and the devp2p sub-project in trinity as well.  Assuming that plugins become a thing in trinity, we'll want enode address information in 3rd party plugins as well as if we implement PSS or SHH protocols as trinity plugins.  Maybe we should have a small `enode` library that implements some basic primatives for working with and validating enodes.

Reply:
This sounds good to me. Is an issue in this codebase the best way to track that?